src/lang/interpreter.cc

```diff
@@ -638,194 +715,467 @@ namespace verona::interpreter
-        // If a behavior is pending, set the successor
-        if (predecessor->status != rt::core::Behavior::Status::Done)
+        // If a behaviour isn't Done, set the successor
+        if (predecessor->status != rt::core::Behaviour::Status::Done)
         {
-          if (predecessor->succ.insert(behavior).second)
+          predecessor->cown_succ[cown] = behaviour;
+          behaviour->cown_ctn += 1;
+          // Only needed for Mermaid:
+          if (predecessor->succ.insert(behaviour).second)
           {
-            behavior->pred_ctn += 1;
+            behaviour->pred_ctn += 1;
           }
-          // Only needed for Mermaid:
-          behavior->cown_deps[cown] = predecessor.get();
+          behaviour->cown_deps[cown] = predecessor.get();
         }
       }
-      // Update pointer to the last pending behavior
-      this->cowns[cown] = behavior;
+      // Update pointer to the last pending behaviour
+      this->cowns[cown] = behaviour;
     }
 
     std::stringstream ss;
-    if (behavior->pred_ctn == 0)
+    if (behaviour->pred_ctn == 0)
     {
-      this->ready.push_back(behavior);
-      behavior->status = rt::core::Behavior::Status::Ready;
-      ss << "New behavior `" << behavior->get_name() << "` is ready";
+      this->ready.push_back(behaviour);
+      behaviour->status = rt::core::Behaviour::Status::Ready;
+      ss << "New behaviour " << format_behaviour_name(behaviour->get_name()) << " is ready";
     }
     else
     {
-      behavior->status = rt::core::Behavior::Status::Pending;
-      ss << "New behavior `" << behavior->get_name() << "` is pending";
+      behaviour->status = rt::core::Behaviour::Status::Pending;
+      ss << "New behaviour " << format_behaviour_name(behaviour->get_name()) << " is pending";
     }
 
+    // Two solutions:
+    // 1. Draw the schedule here and make sure that ExecSchedule doesn't
+    //    draw the scheudle
+    // 2. Store the message but use it explicitly
     this->next_schedule_msg = ss.str();
-    if (this->current_int)
+    //draw_schedule(ss.str());
+    //std::cout << "!!! " << "Scheduled `" << format_behaviour_name(behaviour->get_name()) << "`" << std::endl;
+  }
+
+  void print_help()
+  {
+    std::cout << "Commands:" << std::endl;
+    std::cout << "- <b>      : Run behaviour number b until the next break point" << std::endl;
+    std::cout << "- s<b>,<n> : Run behaviour number b n step (default n = 0) [Default]" << std::endl;
+    std::cout << "- h        : Prints this message " << std::endl << std::endl;
+  }
+
+
+  std::string processLines(const std::string& input, const size_t desired_lines) {
+    std::istringstream iss(input);
+    std::string line;
+    
+    // Split input into lines
+    size_t i{0};
+    std::string result = "";
+    while (std::getline(iss, line) && i < desired_lines) 
     {
-      this->current_int->pause();
+      result += line + "\n";
+      i++;
     }
+    if (std::getline(iss, line))
+    {
+      result += "   {...}\n";
```

Comment:
```suggestion
      result += "   [...]\n";
```

It might be cool to get the indent from the line to put the `[...]` at the right indention